tox.ini

```diff
@@ -4,8 +4,8 @@ skipsdist = True
 
 [testenv]
 commands =
-     {envpython} -m flake8 .
      {envpython} -m unittest  discover unit_tests
+     {envpython} -m flake8 --exclude build,.tox .
```

Comment:
I changed the order here, since otherwise it's hard to insert temporary debug statements in the code.